galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/RunQuery.java

```diff
@@ -139,13 +139,13 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
       }
 
 
-      runs.sort(Comparator.nullsLast(Comparator.comparing(RunResult::getEnd, Comparator.nullsLast(Comparator.naturalOrder()))));
+      runs.sort(Comparator.nullsLast(Comparator.comparing((RunResult run) -> run.getTestStructure().getEndTime(), Comparator.nullsLast(Comparator.naturalOrder()))));
```

Comment:
please dont use lamda unless necessary.   it makes the code hard to read and maintain.   Create an instream class to do it.
Lamda is great for quickly writing code,  but it is not great for readability, maintainability and code coverage